bin/worker.js

```diff
@@ -31,11 +33,14 @@ export default {
       return new Response('Missing required fields', { status: 400 })
     }
 
+    // Timestamp to measure file retrieval performance
+    const t1 = performance.now()
     const { response, cacheMiss, contentLength } = await retrieveFile(
       BASE_URL,
       pieceCid,
       env.CACHE_TTL,
     )
+    const ttfb = performance.now() - t1
```

Comment:
I think we should measure TTFB from the time our handler (worker) started. That's closer to the TTFB observed by end users.